datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -7387,3 +7389,58 @@ FROM (VALUES ('a'), ('d'), ('c'), ('a')) t(a_varchar);
 query error Error during planning: ORDER BY and WITHIN GROUP clauses cannot be used together in the same aggregate function
 SELECT array_agg(a_varchar order by a_varchar) WITHIN GROUP (ORDER BY a_varchar)
 FROM (VALUES ('a'), ('d'), ('c'), ('a')) t(a_varchar);
+
+# distinct average
+statement ok
+create table distinct_avg (a int, b int) as values
+  (null, null),
+  (1, 1),
+  (2, 2),
+  (3, 3),
+  (4, 4),
```

Comment:
Could you update this test so:
1. The input isn't in order 
2. Add a test for floating point values
3. Test for an input that includes at least one null value
4. the values in `b` are different than the values in `b`